arclightning_frontend/games.html

```diff
@@ -1,67 +1,99 @@
+                { name: "Super Smash Bros", description: "crossover fighting game", genre: ["fighter", "crossover"], thumbnail_path: 'images/peach.png', exe_path: 'C:\\Users\\THISUSER\\SMASHBROS_PATH' },
+                { name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH' },
+                { name: "Touhou", description: "bullet hell with waifus", genre: ["bullet hell", "anime"], thumbnail_path: 'images/touhouWaifu.png', exe_path: 'C:\\Users\\THISUSER\\TOUHOU_PATH' },
 
-                    {name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH'},
-                    {name: "Super Smash Bros", description: "crossover fighting game", genre: ["fighter", "crossover"], thumbnail_path: 'images/peach.png', exe_path: 'C:\\Users\\THISUSER\\SMASHBROS_PATH'},
-                    {name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH'},
-                    {name: "Touhou", description: "bullet hell with waifus", genre: ["bullet hell", "anime"], thumbnail_path: 'images/touhouWaifu.png', exe_path: 'C:\\Users\\THISUSER\\TOUHOU_PATH'},
+                { name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH' },
+                { name: "Super Smash Bros", description: "crossover fighting game", genre: ["fighter", "crossover"], thumbnail_path: 'images/peach.png', exe_path: 'C:\\Users\\THISUSER\\SMASHBROS_PATH' },
+                { name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH' },
+                { name: "Touhou", description: "bullet hell with waifus", genre: ["bullet hell", "anime"], thumbnail_path: 'images/touhouWaifu.png', exe_path: 'C:\\Users\\THISUSER\\TOUHOU_PATH' }
 
-	]
-            },
-        })
-        var keypress = new Vue({
-                el: '#keypress',
-                mounted: function () {
-                    document.addEventListener("keyup", (e) => {
-                        // redirect to demonstration.html if 'L' key is pressed
-                        if (e.keyCode == 76) { window.location.href = 'demonstration.html' }
-                    })
-                }
+            ]
+        },
+        methods: {
+            qrGen: function (hostname) {
+                this.qrStr = '', this.ranHex = ''
+                for (var i = 0; i < 64; i++) { this.ranHex += parseInt(window.crypto.getRandomValues(new Uint32Array(1))[0] / 0x100000000 * 16, 10).toString(16) }
```

Comment:
I think there's a better way to do this. You can create a Uint8Array of size 32 and call window.crypto.getRandomValues to fill it with random values, then use something like https://stackoverflow.com/questions/40031688/javascript-arraybuffer-to-hex to convert the array to hex

Reply:
You're right, that is much better. Changed to:
`this.ranHex = Array.prototype.map.call(window.crypto.getRandomValues(new Uint8Array(32)), x => ('00' + x.toString(16)).slice(-2)).join('')`

Reply:
this is freaking awesome